freebase/location.py

```diff
@@ -0,0 +1,30 @@
+import json
+import requests
+import urllib
+
+def lookup(query):
+    """
+    Look up location entity from Freebase given 'query' keywords.
+    Return list of locations that have the highest score given search query.
+    """
+    service_url = 'https://www.googleapis.com/freebase/v1/search'
+    
+    ## These are the params that go into the get request
+    params = {
+        'query': query,
+        'filter':'(any type:/location/location)'    # filter lets us specify which property to filter our search results on. See Freebase Metasearch.
+    }
+    
+    url = service_url + '?' + urllib.urlencode(params)
+    response = json.loads(requests.get(url).text)
```

Comment:
opt: Can pass `requests.get` parameters directly, `requests.get(service_url, params=params)` http://docs.python-requests.org/en/latest/user/quickstart/#passing-parameters-in-urls